package.json

```diff
@@ -8,7 +8,7 @@
     "engines": {
         "vscode": "^1.74.0"
     },
-    "main": "./out/extension",
+    "main": "./dist/extension",
```

Comment:
is there a point in changing from `out` to `dist`?

Reply:
Conventions